cypress/e2e/navigation.cy.js

```diff
@@ -7,6 +7,7 @@ describe("Navigation bar", () => {
 
     const expected = [
         ["Deputy details", "/supervision/deputies/1"],
+        // ["Contacts", "/supervision/deputies/1/contacts"],
```

Comment:
Do you want this to be commented out?

Reply:
Yes, these are commented out as requested by the ticket, to hide the contacts tab when we go live with this work (until the remaining tickets to view & manage deputy contacts are all completed)